{{ if .Values.clustering_enabled }}
---
# This peer auth policy prevents the second node from failing to talk to the first node instantly.
# Instead, as of this writing, it fails over the course of a timeout. In both cases, it gets the same root error,
# the cluster authentication attempt fails because the second node (`confluence-1`) receives unexpected bytes back
# from the first node (`confluence-0`).
apiVersion: security.istio.io/v1beta1
kind: PeerAuthentication
metadata:
  name: "confluence"
  namespace: {{ .Release.Namespace }}
spec:
  mtls:
    mode: STRICT
  portLevelMtls:
    5701:
      mode: DISABLE
  selector:
    matchLabels:
      app.kubernetes.io/name: confluence
{{ end }}
